src/main/java/com/exactpro/th2/infraoperator/Th2CrdController.java

```diff
@@ -27,15 +27,21 @@
 import com.exactpro.th2.infraoperator.spec.strategy.linkresolver.mq.RabbitMQContext;
 import com.exactpro.th2.infraoperator.spec.strategy.redeploy.ContinuousTaskWorker;
 import com.exactpro.th2.infraoperator.spec.strategy.redeploy.tasks.CheckResourceCacheTask;
+import org.apache.logging.log4j.core.LoggerContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class Th2CrdController {
 
-    private static final Logger logger = LoggerFactory.getLogger(Th2CrdController.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(Th2CrdController.class);
 
-    public static void main(String[] args) {
+    private static final Path LOG4J2_PROPERTIES_DEFAULT_PATH = Path.of("var", "th2", "config", "log4j2.properties");
```

Comment:
question: Why would you hardcode the logging configuration here? There is a property we can use to specify where the logging configuration is located. This location is only specific for the docker container - we should configure this location in the Docker file

Reply:
Path to log config was moved to app argument

Reply:
I still don't understand why we need to change the code for that. There are two environment variables used by `service` script that allow us to specify `-Dlog4j2.configurationFile=` with a path to the th2 logging configuration file:

- JAVA_OPTS
- SERVICE_OPTS

Why do we want to modify the code for that?

Reply:
I agree with you but I'm afraid that we will forget about adding environment variable when our team updates any cluster.
Also, previous version has hard-coded path to log.

Reply:
> I agree with you but I'm afraid that we will forget about adding environment variable when our team updates any cluster.

The env variable is set in when the container is built. How can you forget to add it?

> Also, previous version has hard-coded path to log.

And the current version also has the hard-coded path to the logging configuration. I don't see any difference from the version in the `master` branch...


I am resolving this conversation as I don't see any point in explaining myself one more time.